estuary_updater/handlers/koji.py

```diff
@@ -0,0 +1,90 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+from estuary.models.koji import KojiBuild, KojiTag
+
+from estuary_updater.handlers.base import BaseHandler
+from estuary_updater import log
+
+
+class KojiHandler(BaseHandler):
+    """A handler for Koji related messages."""
+
+    @staticmethod
+    def can_handle(msg):
+        """
+        Determine if this handler can handle this message.
+
+        :param dict msg: a message to be analyzed
+        :return: a bool based on if the handler can handle this kind of message
+        :rtype: bool
+        """
+        supported_topics = [
+            '/topic/VirtualTopic.eng.brew.build.complete',
+            '/topic/VirtualTopic.eng.brew.build.tag',
+            '/topic/VirtualTopic.eng.brew.build.untag'
+        ]
+        return msg['topic'] in supported_topics
+
+    def handle(self, msg):
+        """
+        Handle a message and update Neo4j if necessary.
+
+        :param dict msg: a message to be processed
+        """
+        topic = msg['topic']
+
+        if topic == '/topic/VirtualTopic.eng.brew.build.complete':
+            self.build_complete_handler(msg)
+        elif topic == '/topic/VirtualTopic.eng.brew.build.tag' or \
```

Comment:
This will always evaluate to `True`. You want to do:
```python
elif topic == '/topic/VirtualTopic.eng.brew.build.tag' or topic == '/topic/VirtualTopic.eng.brew.build.untag':
```